docs-ref-services/servicebus.md

```diff
@@ -17,131 +17,77 @@ ms.service: service-bus
-            queueClient.sendAsync(new Message("" + i)).thenRunAsync(() -> { Log.log("Sent message #%d.", j);});
-        }
-
-        while(totalReceived != totalSend) {
-            Thread.sleep(1000);
-        }
-
-        Log.log("Received all messages, exiting the sample.");
-        Log.log("Closing queue client.");
-        queueClient.close();
-    }
-
-    static class MessageHandler implements IMessageHandler {
-        private IQueueClient client;
-
-        public MessageHandler(IQueueClient client) {
-            this.client = client;
-        }
-
-        @Override
-        public CompletableFuture<Void> onMessageAsync(IMessage iMessage) {
-            Log.log("Received message with sq#: %d and lock token: %s.", iMessage.getSequenceNumber(), iMessage.getLockToken());
-            return this.client.completeAsync(iMessage.getLockToken()).thenRunAsync(() -> {
-                Log.log("Completed message sq#: %d and locktoken: %s", iMessage.getSequenceNumber(), iMessage.getLockToken());
-                totalReceived++;
-            });
-        }
-
-        @Override
-        public void notifyException(Throwable throwable, ExceptionPhase exceptionPhase) {
-            Log.log(exceptionPhase + "-" + throwable.getMessage());
-        }
-    }
-}
-```
-
 > [!div class="nextstepaction"]
-> [Explore the Client APIs](/java/api/overview/azure/servicebus/client)
-> [Find more examples here (See also above for more details)](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/)
+> [Explore the Management APIs](/java/api/overview/azure/servicebus/management)
 
-## Management API
+## Libraries for data access
 
-Create and manage namespaces, topics, queues, and subscriptions with the management API.
+The official Service Bus client is available in [source code form on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/) and
+binaries and packaged sources [are available on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22azure-messaging-servicebus%22).
 
-**Please find some examples here:**
-* [Manage Service Bus queues](https://github.com/Azure-Samples/service-bus-java-manage-queue-with-basic-features)
-* [Create and subscribe to Service Bus topics](https://github.com/Azure-Samples/service-bus-java-manage-publish-subscribe-with-basic-features)
+**The [sample code repository](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus) contains samples for:**
+* How to use the [SendMessage](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendMessageAsyncSample.java)
+* How to use the [Processor to receive](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusProcessorSample.java)
```

Comment:
We don;t need to list these links for samples as they are already available in the table below